doc/helpers/command-extensions.md

```diff
@@ -8,28 +8,31 @@ Provides Command/CommandParameter attached properties for common scenarios.
 
 ## Attached Properties
 
-| Property           | Type       | Description                                                     |
-|--------------------|------------|-----------------------------------------------------------------|
+| Property           | Type       | Description                                                      |
+|--------------------|------------|------------------------------------------------------------------|
 | `Command`          | `ICommand` | Sets the command to execute when interacting with the control.\* |
-| `CommandParameter` | `object`   | Sets the parameter to pass to the Command property.             |
+| `CommandParameter` | `object`   | Sets the parameter to pass to the Command property.              |
 
 ### Remarks
 
 - `Command` is executed on:
   - `ListViewBase.ItemClick`
+  - `Selector.SelectionChanged` (except for `ListView` which uses `ItemClick`)
   - `NavigationView.ItemInvoked`
   - `ItemsRepeater` item tapped
   - `TextBox` and `PasswordBox` when the Enter key is pressed
   - any `UIElement` tapped
-- For Command, the relevant parameter is also provided for the `CanExecute` and `Execute` call:
-  > Unless CommandParameter is set, which replaces the following.
-  - `TextBox.Text`
-  - `PasswordBox.Password`
+- For `Command`, the relevant parameter is also provided for the `CanExecute` and `Execute` call:
+  > `CommandParameter` can be set, on the item-container or item-template's root for collection-type control, or control itself for other controls, to replace the following.
```

Comment:
```suggestion
  > `CommandParameter` can be set, on the item-container or item-template's root for collection-type controls. It can also be set on the control itself, to replace the following:
```

Reply:
You changed the meaning of the sentence... I want to keep it as is:
```pseudo
if (control is ItemsControl or ItemsRepeater || control.GetProperty("ItemsSource") is { })
	// set cp on item-container or item-template's root
else
	// set cp on control itself
```
It is either that or this, not "can also be".